src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php

```diff
@@ -373,15 +373,15 @@ private function generateItems(array $keys)
      */
     public static function throwOnRequiredClass($class)
     {
-        $e = new \ReflectionException(sprintf('Class %s does not exist', $class));
+        $e = new \ReflectionException("Class $class does not exist");
```

Comment:
"Exception and error message strings must be concatenated using sprintf;" ;) http://symfony.com/doc/current/contributing/code/standards.html